src/Appwrite/Extend/Exception.php

```diff
@@ -83,6 +83,7 @@ class Exception extends \Exception
     public const TEAM_INVALID_SECRET               = 'team_invalid_secret';
     public const TEAM_MEMBERSHIP_MISMATCH          = 'team_membership_mismatch';
     public const TEAM_INVITE_MISMATCH              = 'team_invite_mismatch';
+    public const TEAM_ID_CONFLICT                  = 'team_id_conflict';
```

Comment:
It's always good to take a look at existing code to try and match existing conventions. In this case, for other similar errors, we have `<RESOURCE>_ALREADY_EXISTS` such as `TEAM_INVITE_ALREADY_EXISTS`. So, this should probably be:

```suggestion
    public const TEAM_ALREADY_EXISTS                = 'team_already_exists';
```